docs/hardware/xiaomi-miwifir3.txt

```diff
@@ -0,0 +1,48 @@
+---
+---
+Xiaomi MiWiFi R3
+================
+
+== How to install LibreMesh on the Xiaomi MiWiFi R3
+
+
+This is a pretty good device with lots of memory, one of the last dual band devices easily hackable. Installing LiMe/LEDE/OpenWrt is a bit more work but can be done. Here are the instructions:
+
+Follow these steps:
+
+* First, get ssh/dropbear access: https://wiki.openwrt.org/toh/xiaomi/mir3#to_get_sshdropbear_access (don't forget to downgrade the firmware before)
+
+* Download libremesh rootfs and kernel files from here:
+ - https://mega.nz/#!8A9CUJLQ!gJNC3SfwDdmMZ4y9BXlWp4mcn1FTAU3vWgsVYK2vW4w
```

Comment:
Shouldn't we put the files to the LiMe server?

Reply:
@p4u

Reply:
This files will be soon deprecated because the new target of LEDE is going to be created (PR already sent). So I would let this as a temporay "non-official" firmware provided by a third party (Bruno).